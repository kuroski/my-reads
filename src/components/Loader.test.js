import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Loader from './Loader';

describe('BookShelf Container', () => {
  const build = () => {
    return shallow(<Loader />);
  };

  it('renders Loader Component correctly', () => {
    const wrapper = build();
    const LoaderTree = renderer.create(wrapper[0]).toJSON();
    expect(LoaderTree).toMatchSnapshot();
  });
});
