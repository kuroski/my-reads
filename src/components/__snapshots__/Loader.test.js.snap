// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookShelf Container renders Loader Component correctly 1`] = `null`;
